src/bindings/c/README.md

```diff
@@ -0,0 +1,28 @@
+# OpenVINO C API
+
+OpenVINO C API is a part of OpenVINO Runtime library. 
+This component is responsible for:
+ * Based representations for the objects in OpenVINO - C API provides structures with C code for basical classes inside the OpenVINO, such as "ov_core_t" for "ov::Core". For more information please read [OpenVINO Runtime C API User Guide](./docs/OpenVINO_Runtime_C_API_User_Guide.md).
+ * Based operations representation from OpenVINO - C API contains basical interfaces (read model, inference, properity set and so on) with C code. For more information please read [OpenVINO Runtime C API User Guide](./docs/OpenVINO_Runtime_C_API_User_Guide.md).
```

Comment:
The second link to `OpenVINO_Runtime_C_API_User_Guide` moreover file has another name